Doc/library/functions.rst

```diff
@@ -1388,6 +1388,11 @@ are always available.  They are listed here in alphabetical order.
    object allows it.  For example, ``setattr(x, 'foobar', 123)`` is equivalent to
    ``x.foobar = 123``.
 
+   .. impl-detail::
+
+   The attribute name does not have to be a valid identifier, For example,
+   ``setattr(x, "1", 123)``, this is not guaranteed behaviour and may change
+   in the future.
```

Comment:
The second 'sentence' is not grammatically correct English.  (And no 'u' in the American spelling of 'behavior'.)  The first sentence could be seen appears to say that the behavior is more or less guaranteed.  An alternative:
"An implementation may accept attribute names that are not valid identifiers, as in ``setattr(x, "1", 123)``.  But this is not guaranteed."

Reply:
Correct spelling and grammar now, but I recommend waiting a few days for other opinions before rushing to accept my content suggestion.

Reply:
Thank you @terryjreedy, you are a better writer than me. I found the document use [behavior](https://docs.python.org/3/tutorial/introduction.html#numbers) as well as [behaviour](https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects), I kinda confuse which one should we use.